ecosystem/python/sdk/examples/rotate-key.py

```diff
@@ -0,0 +1,105 @@
+import asyncio
+import subprocess
+import time
+
+from aptos_sdk.account import Account, RotationProofChallenge
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.authenticator import Authenticator, MultiEd25519Authenticator
+from aptos_sdk.bcs import Serializer
+from aptos_sdk.async_client import FaucetClient, RestClient
+from aptos_sdk.transactions import (
+    EntryFunction,
+    RawTransaction,
+    Script,
+    ScriptArgument,
+    SignedTransaction,
+    TransactionArgument,
+    TransactionPayload,
+)
+from aptos_sdk.type_tag import StructTag, TypeTag
+
+from .common import FAUCET_URL, NODE_URL
+
+WIDTH = 19
+
+def truncate(address: str) -> str:
+    return address[0:6] + '...' + address[-6:]
+
+def format_account_info(account: Account) -> str:
+    vals = [str(account.address()), account.auth_key(), account.private_key.hex()]
+    return ''.join([truncate(v).ljust(WIDTH, ' ') for v in vals])
+
+async def rotate_auth_key_ed_25519_payload(
+    rest_client: RestClient,
+    from_account: Account,
+    to_account: Account
+) -> TransactionPayload:
+    rotation_proof_challenge = RotationProofChallenge(
+        sequence_number=await rest_client.account_sequence_number(from_account.address()),
+        originator=from_account.address(),
+        current_auth_key=from_account.address(),
```

Comment:
Is there a `auth_key` method or params in Account? Maybe we should use that. 

IIRC I think when the account first created, the `auth_key` and address are the same, but if this is already rotated before, it probably don't match.

Reply:
Good catch!! There is an `auth_key()` method, and it derives it from the private key, so I'll change it to that, yeah!

This is actually code from https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/python/sdk/aptos_sdk/account.py#L144

and

https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/python/sdk/examples/multisig.py#L194

I'll update those in a new PR and tag/attribute you in it!